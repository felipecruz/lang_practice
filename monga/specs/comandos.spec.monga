int[] func (int num) {
    int b, x, c, w;
    if (0) {} else {}

    while (b > c[0]) {}
    while (b > c[0]) {
        b = 1;
    }

    x = w[0];
    func (10);
    w = func (b[0]);

    {
        b = 1;
    }
}
