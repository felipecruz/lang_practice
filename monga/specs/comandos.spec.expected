Checking Func foo
Checking return type.. 
Checking Func func
Get Expression Var Type...
Get Expression Var Type...
Get Expression Var Type...
Get Expression Var Type...
Checking assignment types... 
Get Expression Var Type...
Checking assignment types... 
Get call return type..
Get Expression Var Type...
Checking return type.. 
Get Expression Var Type...
